Documentation/ceph-object-bucket-claim.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Object Bucket Claim CRD
+weight: 2800
+indent: true
+---
+
+# Ceph Object Bucket Claim
```

Comment:
Taking a step back and thinking about this, do we even need this document? If the changes in the other doc (ceph_object.md) are all the user needs to use this feature, then do we need these details? Maybe we do still need to describe the properties of the OBC in more detail, but we also don't need to turn this into a design doc that describes everything. We only need to describe what is needed for the user and we need to avoid details he doesn't need. That's what the design doc is for. :)

Reply:
Yeah, I agree with @travisn, since it has a lot of information than initially planned. 
In the  "Sample" we can only include about obc, storage class and app rest can be IMHO. 
@jeffvance @copejon please provide your valuable feedback.

Reply:
I am updating the version as mentioned above please review